### ISSUE TITLE
[DatePicker] A11y - read the date as current

### ISSUE THREAD

**Issue by adyubina** [mui-org/material-ui-pickers#2110]
## Environment
Win 10/ Chrome/ JAWS 18
MacOS/ Safari/ VoiceOver

- [ x] I have searched the [issues](https://github.com/mui-org/material-ui-pickers/issues) of this repository and believe that this is not a duplicate.

## Summary 💡
We use DatePicker component in our app and in the latest a11y report it was stated that when the current date is selected the screen reader doesn't read it as current

![8474](https://user-images.githubusercontent.com/70582404/93086317-3bae6180-f697-11ea-85c8-14610b7c28c6.png)

## Examples 🌈
The proposed variant is to add the phrase "current date" between the date and the time.

## Motivation 🔦

Such a addition will help screen readers' users to distinguish the current date from any other.

**Comment by oliviertassinari** [comment]
@adyubina Are you using the latest version http://next.material-ui-pickers.dev/?

**Comment by dmtrKovalenko** [comment]
Check the http://next.material-ui-pickers.dev :)

**Comment by adyubina** [comment]
thank you @oliviertassinari and @dmtrKovalenko 
I tried http://next.material-ui-pickers.dev but there the problem is present as well
Please see my video and screenshots attached
[IMG_3203.mov.zip](https://github.com/mui-org/material-ui-pickers/files/5218704/IMG_3203.mov.zip)
![image](https://user-images.githubusercontent.com/70582404/93092890-4b7e7380-f6a0-11ea-9b34-50232bca9b01.png)
![image](https://user-images.githubusercontent.com/70582404/93092791-2f7ad200-f6a0-11ea-8463-ad2c5e2e8eab.png)

Yes, I have noticed that when tabbing from the date input to calendar Icon the current date is read. But it is not the issue we care about. Our case is about when the calendar popup-up is open and a user is tabbing through days. In this case eye-sighted users can see on the screen where "today" is, but there is no information for screen readers to distinguish today from the rest of the days.

**Comment by adyubina** [comment]
Hi @dmtrKovalenko and @oliviertassinari 
have you had a chance to look at the data a posted? It looks like I can't proceed on the in my project without your assistance, so could you please give the issue describe above some of your time.
